src/insns/scbnds_32bit.adoc

```diff
@@ -1,16 +1,17 @@
 <<<
 
-[#CSETBOUNDSIMM,reftext="CSETBOUNDSIMM"]
-==== CSETBOUNDSIMM
+[#SCBNDSI,reftext="SCBNDSI"]
+==== SCBNDSI
 
-See <<CSETBOUNDS>>.
+See <<SCBNDS>>.
 
-[#CSETBOUNDS,reftext="CSETBOUNDS"]
-==== CSETBOUNDS
+[#SCBNDS,reftext="SCBNDS"]
+==== SCBNDS
 
 ifdef::cheri_v9_annotations[]
-NOTE: *CHERI v9 Note:* <<CSETBOUNDS>> was CSETBOUNDSEXACT,
-<<CSETBOUNDSIMM>> would have been CSETBOUNDSEXACTIMM if it had existed.
+NOTE: *CHERI v9 Note:* SCBNDS was called CSETBOUNDS.
+
+NOTE: *CHERI v9 Note:* SCBNDSI would have been CSETBOUNDSEXACTIMM if it had existed.
```

Comment:
The 64-bit compression format basically made it that instruction but agreed that it could have been an issue for 32-bit

Reply:
I'll resolve this - let me know if something does need to change